locales/en/index.yml

```diff
@@ -838,7 +838,7 @@ onboarding:
       invalid: The code is not valid
       match: The code doesn’t match
     policy:
-      headerTitle: "For security reasons, the code must not contain:"
+      headerTitle: "Scegli un codice difficile da individuare e che non contenga:"
```

Comment:
@AliceAzzolini i need the english text! Thanks you so much!

Reply:
```suggestion
      headerTitle: Choose a code that's hard to guess and doesn't contain:
```